kernel/cPrimitives.mli

```diff
@@ -51,11 +51,19 @@ type t =
   | Float64ldshiftexp
   | Float64next_up
   | Float64next_down
+  | Arraymake
+  | Arrayget
+  | Arraydefault
+  | Arrayset
+  | Arraycopy
+  | Arrayreroot
+  | Arraylength
+  | Arraymaxlength
 
 val equal : t -> t -> bool
 
 type arg_kind =
-  | Kparam (* not needed for the elavuation of the primitive*)
+  | Kparam (* not needed for the evaluation of the primitive*)
```

Comment:
Is the comment still true? For instance `Arraymake` uses the type in the literal it produces.

Reply:
It was indeed a bug, should be fixed now.